scripts/cookie-script.js

```diff
@@ -0,0 +1,328 @@
+/* eslint-disable */
```

Comment:
May I suggest to add the cookie script in a separate PR and keep this PR limited to GTM.

Reply:
@sdmcraft  sure, let me update the pr